superset-frontend/src/SqlLab/components/ShareSqlLabQuery/ShareSqlLabQuery.test.tsx

```diff
@@ -72,6 +72,7 @@ const unsavedQueryEditor = {
   schema: 'query_schema_updated',
   sql: 'SELECT * FROM Updated Limit 100',
   autorun: true,
+  templateParams: '{ "my_value": "foo" }',
```

Comment:
@justinpark how are these template parameters actually leveraged in the test? It's not apparent to me.

Reply:
@john-bodley the following block verify that this extra templateParams payload is being passed through api.

https://github.com/apache/superset/blob/09a7d01ea763e1335de9975240e0ba97d0bd22e1/superset-frontend/src/SqlLab/components/ShareSqlLabQuery/ShareSqlLabQuery.test.tsx#L126-L130